src/main/java/icbm/classic/content/blocks/emptower/TileEMPTower.java

```diff
@@ -98,7 +98,7 @@ public void update()
             rotation += rotationDelta;
             if (rotation > 360)
             {
-                rotation = 0;
+                rotation -= 360;
```

Comment:
We should do a divide system here instead. What is likely making it not smooth is that if we did 365 we would snap that 5 back to 0 making for a sharp visual.